addition_wasm/README.md

```diff
@@ -0,0 +1,15 @@
+- [add.h](add.h) is a tfcompile-generated header file
+- [add.cc](add.cc) is used to invoke the tfcompile-generated code using the header file
+- The [objects](objects/) directory contains all necessary compiled dependencies that can be linked to generate a WASM binary file. [add_helper.o](objects/add_helper.o) and [add_model.o](objects/add_model.o) were generated using a modified `tfcompile` with WASM support, and the rest of the files were compiled using a command similar to what Bazel uses to build the x86 binary if one were using the `cc_binary` build macro. These commands can be seen from Bazel using the `--subcommands` option, and a sample command is given below. You may need to remove some flags that Bazel uses in its subcommands if they cause errors when trying to manually compile dependencies with emscripten.
```

Comment:
Future-you will want to know which "some flags" were. IIRC `-fstack-protector` was one. Any others?